### ISSUE TITLE
"Web pages slowing your browser" for particular source script

### ISSUE THREAD

**Issue by gwarser** [Rob--W/crxviewer#80]
Trying to view `src/script/content_script.js` in this extension:

```
https://chrome.google.com/webstore/detail/blocktube/bbeaicapbccfllodepmimpkgecanonai
https://addons.mozilla.org/en-US/firefox/addon/blocktube/
```

```
https://robwu.nl/crxviewer/?crx=https%3A%2F%2Fclients2.google.com%2Fservice%2Fupdate2%2Fcrx%3Fresponse%3Dredirect%26os%3Dlinux%26arch%3Dx86-64%26os_arch%3Dx86-64%26nacl_arch%3Dx86-64%26prod%3Dchromiumcrx%26prodchannel%3Dunknown%26prodversion%3D9999.0.9999.0%26acceptformat%3Dcrx2%2Ccrx3%26x%3Did%253Dbbeaicapbccfllodepmimpkgecanonai%2526uc&qf=manifest.json&qb=1
https://robwu.nl/crxviewer/?crx=https%3A%2F%2Faddons.mozilla.org%2Ffirefox%2Fdownloads%2Flatest%2Fblocktube%2Fplatform%3A2%2Fblocktube.xpi&qf=manifest.json&qb=1
```

both Firefox Nightly and Chromium uses 100% of CPU core and shows message about not responding script. 

Not sure this file is hard to decode on my PC or something "looping" in CRX Viewer.

Firefox 67.0a1 Build ID 20190308093915
Chromium 72.0.3626.121 (Oficjalna wersja) Manjaro Linux (64-bitowa)

***

Hmmm

![image](https://user-images.githubusercontent.com/886325/54076046-00afbe80-42a7-11e9-9953-206dc18cea91.png)

https://perfht.ml/2H8NV9U

**Comment by Rob--W** [comment]
This is a bug in the syntax highlighting library, and appears to have been fixed upstream in https://github.com/PrismJS/prism/commit/ac09e97575f8751389a964d5465497d8d423bf51

I will update Prism.js to the latest version to fix this.

**Comment by Rob--W** [comment]
Published fix in version 1.6.10.
